smartsim/_core/_cli/build.py

```diff
@@ -391,20 +492,46 @@ def execute(args: argparse.Namespace) -> int:
                 onnx,
                 args.torch_dir,
                 args.libtensorflow_dir,
+                modify_python_env=modify_python_env,
                 verbose=verbose,
             )
 
             backends = list(map(str.capitalize, installed_redisai_backends()))
-            logger.info(
-                f"{', '.join(backends) if backends else 'No'} backend(s) built"
-            )
+            logger.info(f"{', '.join(backends) if backends else 'No'} backend(s) built")
 
     except (SetupError, BuildError) as e:
         logger.error(str(e))
         return 1
 
     logger.info("SmartSim build complete!")
-    return 0
+    return verify.execute(args) if args.verify else 0
+
+
+def _check_python_pkgs(
+    with_tf: bool,
+    with_pt: bool,
+    with_onnx: bool,
+    device: _TDeviceStr,
+    fetch_if_missing: bool,
+    verbose: bool,
+) -> None:
+    """Install python ML package dependencies via pip"""
+    if with_onnx:
+        install_py_onnx_version(
+            fetch_if_missing=fetch_if_missing,
+            verbose=verbose,
+        )
+    if with_tf:
+        install_py_tf_version(
+            fetch_if_missing=fetch_if_missing,
+            verbose=verbose,
```

Comment:
I hate passing verbose around. It's not a parameter... it's a cross cutting configuration value. If we did custom log level, couldn't we remove the parameter and just log?

Reply:
Totally agree!!